libnestutil/block_vector.h

```diff
@@ -63,7 +63,8 @@ class bv_iterator
   using iter_ = bv_iterator< value_type_, cv_value_type_&, cv_value_type_* >;
 
   const BlockVector< value_type_ >* block_vector_; //!< BlockVector to which this iterator points
```

Comment:
This is a bit of a nitpick, but moving this comment to make it consistent with the comments below.
```suggestion
  //! BlockVector to which this iterator points
  const BlockVector< value_type_ >* block_vector_;
```